content/browser/renderer_host/render_process_host_impl.cc

```diff
@@ -3205,6 +3214,9 @@ void RenderProcessHostImpl::AppendRendererCommandLine(
     command_line->AppendSwitchASCII(blink::switches::kJavaScriptFlags,
                                     "--jitless");
 
+  // [RecordReplay] Call `IsForRecordReplayRecording()` here and
```

Comment:
Is this a left-over comment?

Reply:
Likely.  I thought I had gotten all of them, will check.